vochain/state/electionprice/electionprice.go

```diff
@@ -164,6 +167,12 @@ func (p *Calculator) Price(params *ElectionParameters) uint64 {
 		return 0
 	}
 
+	// If the election duration is specified in seconds, convert it to blocks
+	// This is a temporary solution until the block duration support is removed.
+	if params.ElectionDurationSeconds > 0 {
```

Comment:
Is this still required?, asking because new created elections will be working with time and not blocks.

Reply:
We still calculate the price in blocks because the formula is based on blocks. Change to seconds would require changing the full formula, and I let this for a future PR (since it requires also simulations and so on).